ivy_models/transformers/perceiver_io.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 # local
-from ivy_models.transformers.helpers import PreNorm, FeedForward
+from models.ivy_models.transformers.helpers import PreNorm, FeedForward
```

Comment:
I think we should generally be running things from inside the models folder, and therefore we shouldn't need to add this in the import path. Maybe this has changed in the `Dockerfile`?

Reply:
yes it's not required while using docker. It was required when testing locally